src/app/components/shared/formly/image-input.component.ts

```diff
@@ -30,18 +27,65 @@ import { asFormControl } from "./helper";
         />
 
         <button
+          *ngIf="!usesDefaultImage"
           type="button"
           (click)="removeImage()"
-          class="btn btn-outline-danger"
-        >Remove</button>
+          class="btn btn-outline-danger pb-1"
+        >
+          Remove
+        </button>
       </div>
     </div>
   `,
 })
-export class ImageInputComponent extends FieldType {
+export class ImageInputComponent extends FieldType implements AfterViewInit {
   @ViewChild("imageInput")
   public imageInput: ElementRef;
   public asFormControl = asFormControl;
+
+  public ngAfterViewInit() {
+    if (!this.usesDefaultImage) {
+      const currentImage = this.model.imageUrls?.shift();
+      const imageFileName = this.fileName(currentImage?.url);
+
+      // if the current model has a different image to the default image, display the name of the file in the image input
+      // for security reasons, modern web browsers don't allow changing a file inputs value directly
+      // because of this we need to use a data transfer which is performed in "protected mode"
+      // https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/files
+      const dataTransfer = new DataTransfer();
+      dataTransfer.items.add(new File([""], imageFileName));
+      this.imageInput.nativeElement.files = dataTransfer.files;
+    }
+  }
+
+  /**
+   * A predicate that returns if the models image is a default image
+   */
+  public get usesDefaultImage(): boolean {
+    const imageUrls = this.model?.imageUrls;
+    const defaultImageCounts = imageUrls?.filter(
```

Comment:
maybe use array.some() to test for the presence of a default image in the array, for succinctness and efficiency